src/object_functionality.js

```diff
@@ -0,0 +1,54 @@
+//Creating a car class
+class car {
+  constructor(manufacturer, model, year, type, price) {
+    this.manufacturer = manufacturer;
+    this.model = model;
+    this.year = year;
+    this.type = type;
+    this.price = price;
+  }
+
+  toString() {
+    return `
+      Manufacturer: ${this.manufacturer}
+      Model: ${this.model}
+      Year: ${this.year}
+      Type: ${this.type}
+      Price: ${this.price}
+      `;
+  }
+}
+
+//Creating new cars
+car1 = new car("BMW", "M5", "2020", "Sedan", "80,000");
+car2 = new car("BMW", "M4", "2019", "Coupe", "50,000");
```

Comment:
This naming is fine for the example, but we would probably use more descriptive names in a real program, e.g. `bmwM4` and `bmwM5` or something like that.